guides/vsf.md

```diff
@@ -1,147 +1,78 @@
 
-### package.json
-
-Update the xdn:build and xdn:deploy scripts as follows:
-
-```js
-"xdn:build": "xdn build && cp -r lang .xdn/lambda/lang",
-"xdn:deploy": "yarn xdn:build && xdn deploy --skip-build platform",
+```
+npm i -g @vue-storefront/cli@next
+vsf init <project-name>
+cd <project-name>
+npm install
+npm install --save @vue-storefront/commercetools@latest @vue-storefront/nuxt@latest @vue-storefront/nuxt-theme@latest
 ```
 
-### nuxt.config.js
+## Vue Storefront bugfixes
 
-Make the following changes to `nuxt.config.js`:
+The VSF team is still working on the next version of VSF, and there are a few bugs.
 
-1. Move all entries in `modules` to `buildModules`
-2. Remove `@nuxtjs/pwa` from `buildModules`. It is not needed because `@xdn/nuxt/module` builds and injects its own service worker.
-3. Add `@xdn/nuxt/module` to `buildModules`
-4. Ensure that loading `nuxt.config.js` does not fail when webpack is not present.
+In order to get the boilerplate project building, please make these changes:
 
-In nuxt.config.js, replace this:
+In `composables/useUiHelpers/index.ts`, remove this line:
 
 ```js
-require('dotenv').config()
-const webpack = require('webpack')
+import { FacetSearchInput } from '@vue-storefront/commercetools'
 ```
 
-with this:
+and change this line:
 
 ```js
-let webpack
-
-try {
-  require('dotenv').config()
-  webpack = require('webpack')
-} catch (e) {
-  // will get here in the cloud, but we can just catch this error because webpack is only needed during the build phase
-}
+const getFacetsFromURL = (): FacetSearchInput => {
 ```
 
-And replace this:
+to
 
 ```js
-build: {
-  transpile: ['vee-validate/dist/rules'],
-  plugins: [
-    new webpack.DefinePlugin({
-      'process.VERSION': JSON.stringify({
-        // eslint-disable-next-line global-require
-        version: require('./package.json').version,
-        lastCommit: process.env.LAST_COMMIT || ''
-      })
-    })
-  ]
-}
+const getFacetsFromURL = (): any => {
 ```
 
-with this:
+## Install the XDN CLI
 
-```js
-build: webpack && {
-  transpile: ['vee-validate/dist/rules'],
-  plugins: [
-    new webpack.DefinePlugin({
-      'process.VERSION': JSON.stringify({
-        // eslint-disable-next-line global-require
-        version: require('./package.json').version,
-        lastCommit: process.env.LAST_COMMIT || '',
-      }),
-    }),
-  ],
-}
-```
-
-### tsconfig.json
+To prepare your Vue Storefront app for deployment on the Moovweb XDN, run the following commands in the root folder of your project:
 
-In tsconfig.json, add `"sw"` to `exclude`.
+```
+npm install -g @xdn/cli
+npm i --save-dev @xdn/cli@2.27.1-next-1604428707-ccdfc873.0
+xdn init --xdnVersion 2.27.1-next-1604428707-ccdfc873.0
+```
 
-### routes.js
+### nuxt.config.js
 
-Update the XDN router located in `packages/<platform>/theme/routes.js`:
+The XDN init command should have automatically moved all your `modules` to `buildModules` in order to deploy the smallest possible build to the XDN.
```

Comment:
This is not exactly the case. I still have:

```js
 modules: [
    'nuxt-i18n',
    'cookie-universal-nuxt',
    'vue-scrollto/nuxt'
  ],
```

Is that intended? If so, please update the docs.